src/pipeline_v3/cli/utils/validation.py

```
@@ -7,6 +7,8 @@
 from pathlib import Path
 from typing import Any
 
+from utils.security import PathSecurityValidator, SecurityError
```

Pylint (3.3.7)

[error] Unable to import 'utils.security'

(E0401)